src/utilities/trim_projdata.cxx

```diff
@@ -0,0 +1,246 @@
+//
+//
+/*
+    
+*/
+/*!
+
+  \file
+  \ingroup utilities
+  \brief Main program for trim_projdata
+
+  \author Parisa Khateri
+
+
+   \par Usage:
+   \code
+   trim_projdata [-t num_tang_poss_to_trim] \
+        output_filename input_projdata_name 
+   \endcode
+   \param num_tang_poss_to_trim has to be smaller than the available number
+      of tangential positions.
```

Comment:
It's not clear from this documentation which tangential positions are trimmed, min or max tangential positions?

Reply:
The trim should be symmetric. I can include in the documentation. 
Yes, two utilities should be included in the CMakeLists indeed. just ignored to be tracked locally. I'll do it.